epub33/core/index.html

```diff
@@ -8256,6 +8256,8 @@ <h5>The <code>text</code> element</h5>
 								href="https://www.w3.org/TR/SVG11/paths.html">paths</a>, <a
 								href="https://www.w3.org/TR/SVG11/shapes.html">basic shapes</a>, or <a
 								href="https://www.w3.org/TR/SVG11/text.html">text</a> elements in SVG). </p>
+						
+						<p class="note">While the <code>src</code> attribute of a <code>text</code> element may refer to embedded timed media, such as an [[html]] [^video^] element, this is atypical usage and, as Reading System behavior is unspecified, may produce unpredictable results.</p>
```

Comment:
This is still kind of bugging me this morning. I wonder if we should be specific that we don't expect playback anymore, and that authors should consider informing users that audio/video material is available.

```suggestion
						<p class="note">While the <code>src</code> attribute of a <code>text</code> element may refer to embedded timed media, such as an [[html]] [^video^] element, such use is atypical for synchronized audio playback. Reading System are not expected to initiate playback of the media. When referencing such media, authors should consider informing users in the synchronized audio that a resource is available for playback.</p>
```

Reply:
I don't know, it feels out of place to me for the spec to be giving best practice advice for what a book's contents should say.

Reply:
"should consider" in an informal note is a fairly weak statement, ie, a weak form of best practice advice. I am not bothered by this.

I am mildly bothered by another point, though. I believe it is formally o.k. to say

```xml
<text src="link_to_a_section_in_a_content"/>
<text src="link_to_a_video"/>
```

Ie, MO is not, strictly speaking, for audio playback only; I could use a video playback, too. Otherwise, the very term "Media Overlay" would be a misnomer. I realize audio is _the_ major use case, and maybe no RS would be able to do anything with the structure above (is it worth a test?) and I am not sure what the a11y use case would be, but our spec is allowing it... Hence, the "for synchronized audio playback" seems to be out of place.

Reply:
(B.t.w, if this usage is valid and has a minimum usability, we should probably take over the SMIL `<video>` element... in a future release.

Reply:
@iherman well you could have a `par` containing either of your `text` element examples but you cannot have both of those `text`s in parallel with each other.

Reply:
Hm. What this means is that I cannot use a video for playback. Ie, if we need it, we need a `<video>` element (yes, that is for a future version, if any...). But this means is that "Media Overlay" is really a misnomer. Is it too late or too awkward to rename the full functionality to "Audio Overlay" or "Audio playback". This is really what we are talking about, isn't?

Reply:
You could still have this sort of thing:

```
<par>
  <text src="link_to_a_video"/>
</par>
<par>
  <text src="link_to_html_para"/>
  <audio src="audio_file".../>
</par>
...
```

The changes in my PR have not in any way affected an author's ability to use MO in this way. I think what you were asking about (two texts in parallel, one of them an embedded video) has never been possible as a `par` must have exactly one `text` child. 

The PR changes do affect what a RS may do when it would encounter the above markup as we are now leaving it entirely up to implementations.

Regarding renaming MO ... I don't think we should do that. Not only would it be confusing for consumers of the spec, to have a major feature take on a new name but do exactly the same thing; it closes the door that we have been intentionally leaving open to potential future expansion of media types (that was the original reason for "media" and not "audio" overlays). 

Although, between you and me and anyone reading this thread, I am hoping that, in EPUB's future, we can adopt some of the VTT-based work in the [Sync Media CG](https://w3c.github.io/sync-media-pub/#with-webvtt) rather than make an increasingly hard to implement SMIL flavor.

Reply:
What this whole thread and discussion tells me is that there is a need to have a separate reflection and plan for the future evolution of MO or anything replacing it. But that is definitely not for the current PR nor the Working Group as it stands today. Plan to participate (remotely) at the Publishing Salon at TPAC, @marisademeglio ?

Reply:
I may be able to join for a bit -- do you think there will be time and interest to discuss the future of MO at the Publishing Salon?

Reply:
Something to check with @wareid and @TzviyaSiegman !

Reply:
However, this very issue/PR may be on the F2F meeting's agenda, so it would be important for us that you would be there!

Reply:
We don't need the bit about announcing that audio/video is there. What bothered me more was that we are saying the behaviour is unspecified when we're deprecating playback. We should be more direct about the change so authors know that it's not something that will get specified later.

I've trimmed the suggested text above to focus on that part only.

Reply:
I like it! At the start, where it says:

> Reading system playback of timed media is now deprecated.

Could it instead be:

"Guidance for reading system playback of timed media is now deprecated."

?

Reply:
I've switched the suggestion to: "[[epub-rs-33]] no longer provides guidance for reading systems on the playback of timed media ..."

Does that work better?

Reply:
Ok final adjustment, I promise! How about 

"<p class="note">[[epub-rs-33]] no longer provides guidance for reading systems on the playback of embedded timed media. This scenario arises when a Media Overlays <code>text</code> element refers to an [[html]] [^video^] element. While this usage is permitted in Media Overlays, reading systems behavior is implementation-specific.</p>"